### ISSUE TITLE
Allow tabs in DocTestParser

### ISSUE THREAD

**Issue by jacksonllee** [cjw296/sybil#24]
Hello, thank you very much for this nifty library! In my use case, there are non-Python blocks that contain tabs but these tabs should just be ignored. My workaround is to let `DocTestParser` take an `allow_tabs` kwarg in my fork:

https://github.com/jacksonllee/sybil/commit/68944ab9881c8af70254317f36ae996e32faf25c

If there's any interest, I'd be more than happy to contribute a pull request so that it's available in the upstream source here as well.

**Comment by cjw296** [comment]
Can you share some examples of these non-Python blocks?
If they're not Python, how does doctest evaluate them?

**Comment by jacksonllee** [comment]
> Can you share some examples of these non-Python blocks?

Absolutely. I use the reStructuredText block to illustrate a text format that happens to use tabs. Here's an (actual) example ([source](https://github.com/jacksonllee/pycantonese/blob/0156e04031ab6660fd2bacfd8802951d567fe7b2/docs/source/reader.rst)):

```rst
To illustrate, let us consider the following CHAT utterance with its ``%mor``
tier::

    *XXA:	喂 遲 啲 去 唔 去 旅行 啊 ?
    %mor:	e|wai3 a|ci4 u|di1 v|heoi3 d|m4 v|heoi3 vn|leoi5hang4 y|aa3 ?
```

> If they're not Python, how does doctest evaluate them?

I'm not sure -- I haven't dug into the internals of doctest / Sybil too much. For my use case. I'd want doctest / Sybil to simply ignore blocks like this, which the [commit I referred to above](https://github.com/cjw296/sybil/issues/24#issue-752703639) would allow when `allow_tabs=True` in `DocTestParser`. Would there by any chance be a better way to accomplish this that I've missed?

**Comment by cjw296** [comment]
Ah, so the problem is actually that doctest is spotting tabs in blocks it won't even match...

**Comment by cjw296** [comment]
43445d80988f3c70dc91edbf1e8f746f6061497e should resolve this, it'll be in a 2.0.1 release as soon as CI finishes running.

**Comment by jacksonllee** [comment]
Thank you for the quick fix! I've just upgraded to Sybil 2.0.1 and my CI also passed as well. Thanks again!
